py-geopolars/python/geopolars/io/file.py

```diff
@@ -0,0 +1,20 @@
+import polars
+
+from geopolars.internals.geodataframe import GeoDataFrame
+
+try:
+    import geopandas
+except ImportError:
+    geopandas = None
+
+
+def read_file(*args, **kwargs):
+    if geopandas is None:
+        raise ImportError(
+            "Geopandas is currently required for the read_file method. Install it with `pip install geopandas`."
+        )
+
+    geopandas_gdf = geopandas.read_file(*args, **kwargs)
+    arrow_table = geopandas.io.arrow._geopandas_to_arrow(geopandas_gdf)
```

Comment:
This makes me think we should promote this function to part of the public GeoPandas API as something like a `to_arrow` method.  @ jorisvandenbossche?

Reply:
A public API to import/export geopandas to in-memory arrow data would be useful.